controllers/metrics.go

```diff
@@ -102,8 +152,50 @@ func DeleteDRPolicySyncIntervalMetrics(labels prometheus.Labels) bool {
 	return dRPolicySyncInterval.Delete(labels)
 }
 
+// lastSyncDuration Metrics
+func SyncDurationMetricLables(drPolicy *rmn.DRPolicy, drpc *rmn.DRPlacementControl) prometheus.Labels {
```

Comment:
Lables- you mean labels?

Reply:
I have corrected the typos, there were a few others besides this. thanks for pointing it out.